Lib/tkinter/__init__.py

```diff
@@ -1045,8 +1045,16 @@ def winfo_reqwidth(self):
         return self.tk.getint(
             self.tk.call('winfo', 'reqwidth', self._w))
     def winfo_rgb(self, color):
-        """Return tuple of decimal values for red, green, blue for
-        COLOR in this widget."""
+        """Return a tuple of the RGB values for color in this widget.
```

Comment:
This docstring looks too verbose. And its style differs from the style of other docstrings too much.

The docstring is just a brief reminder about what the function does. The module documentation can be more detailed.